package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "week5_api_capstone",
+  "version": "1.0.0",
+  "description": "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\r https://github.com/paulshaun-kdtk/Week5_API_Capstone",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
```

Comment:
Please ensure to set up the necessary command to enable the test command to execute your written tests.